frontend/app/src/components/accounts/management/types/AgnosticAddressAccountForm.vue

```diff
@@ -40,12 +39,11 @@ const label = computed<string>({
   },
   set(label: string) {
     const model = get(modelValue);
-    const labelData = label ? { label } : {};
     set(modelValue, {
       ...model,
       data: {
-        ...objectOmit(model.data, ['label']),
-        ...labelData,
+        ...model.data,
+        label,
```

Comment:
this I guess needs the changes to work right? and we are sending the label either way?
Do we send the `` label too?